src/main/java/frc/robot/OperatorPanel.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.commands.AcquireCommand;
+import frc.robot.constants.IOConstants;
+import frc.robot.libraries.ControlPanel1038;
+
+public class OperatorPanel extends ControlPanel1038 {
```

Comment:
Since we are never going to re-use `ControlPanel1038`, no need to make a class here. Just have this extend `GenericHID` and do it all in here.